### PR TITLE
UHF-9829 Main news articles

### PR DESCRIPTION
**Pull request by khalima**
# [UHF-9829](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9829)

**Notice!** Check and merge the [news article PR](https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/572) before merging this one.

## What was done

- Moved the "ordered news" views admin page styles to etusivu-instance.
- Created the "ordered news articles list" draggable view and added a menu link for it in to the tools menu.
- Created/Updated translations for the news article promoted field, news articles, front page top news paragraph, ordered news list view and ordered news articles list view.
- Added "main news articles" block to "frontpage news" view and added "news article" content type to "of interest" view.

## How to install

* Make sure your helfi-etusivu instance is up and running on latest dev branch.
    * `git fetch --all && git checkout UHF-9736 && git pull origin UHF-9736`
    * `make fresh`
* Switch to `UHF-9829_main_news_articles` branch in etusivu instance and run `drush deploy`
* Update the HDBT Admin theme
    * `composer require drupal/hdbt_admin:dev-UHF-9829_main_news_articles`
* Run `make drush-cim drush-cr`

## How to test

* [x] Go to https://helfi-etusivu.docker.so/fi/admin/ordered-news and https://helfi-etusivu.docker.so/fi/admin/ordered-news-articles. Check that the first three or four rows are coloured with green backgrounds
* [x] Check that code follows our standards

## Continuous documentation

* [ ] This feature has been documented/the documentation has been updated
* [x] This change doesn't require updates to the documentation

## Translations

* [x] Translations have been added to .po -files and included in this PR

[UHF-9829]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9829?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: public/modules/custom/helfi_etusivu/assets/css/views.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+/* Main news article feed ordering -view highlight. */
+#views-form-ordered-news-articles-list-ordered-news-articles {
+  table tbody tr {
+    &:nth-child(-n+3) {
+      background: rgba(46,160,67,0.1);
+    }
+    &:nth-child(3) {
+      border-bottom: 1px solid rgba(0,0,0,0);
```
Comment (teroelonen) [inline public/modules/custom/helfi_etusivu/assets/css/views.css:20]: This makes the border transparent. It's not the idea here maybe?

File: public/modules/custom/helfi_etusivu/assets/css/views.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* Main news feed ordering -view highlight. */
+#views-form-ordered-news-list-ordered-news {
+  table tbody tr {
+    &:nth-child(-n+4) {
+      background: rgba(46,160,67,0.1);
+    }
+    &:nth-child(4) {
+      border-bottom: 1px solid rgba(0,0,0,0);
```
Comment (teroelonen) [inline public/modules/custom/helfi_etusivu/assets/css/views.css:8]: This makes the border transparent. It's not the idea here maybe?

**Review by teroelonen — CHANGES_REQUESTED** [review]
Works well but some issues with the border color on the admin UI still.

File: public/modules/custom/helfi_etusivu/assets/css/views.css (hunk 2 above)
Comment (khalima) [reply public/modules/custom/helfi_etusivu/assets/css/views.css]: Apparently the original styles were changed along the way. I removed the border styles and added hover styles for the table row.

**Review by teroelonen — APPROVED** [review]
This works well now and looks good 🦖

**Pull request merged**